packages/frontend/core/src/modules/i18n/services/i18n.ts

```diff
@@ -0,0 +1,33 @@
+import { getI18n } from '@affine/i18n';
+import { Service } from '@toeverything/infra';
+
+import type { I18nFuncs, I18nString } from '../types/i18n';
+
+class RawI18nService extends Service {
```

Comment:
1. shall we move out i18n refactor out of this pr?
2. it seems to me i18n does not need to be scoped into DI, since i18n is already a global singleton

Reply:
I want to use i18n outside of react, and I wrote this service, I think it would be ok to use the global instance.

I can write some utility functions in @affine/i18n

Reply:
yeah I think the react part of i18n is already a wrapper around the i18n global instance. It is indeed useful to be used outside of react, including to be used somehow in lit